### ISSUE TITLE
jsfight.club opening day announcement (Javascript Battle, group 4)

### ISSUE THREAD

**Issue by Asuza** [JSJitsu/hero-starter#17]
_TLDR; https://jsfight.club brings back everything you loved about JavascriptBattle.com. Join the fray!_

**Welcome, everyone!**

Over a year ago, I found out about JavascriptBattle.com. I quickly became obsessed with trying to make my AI be the best it could be, spending hours tweaking my code and running test battles against myself. I stayed with it for several weeks, until the site no longer loaded. Eventually, it was stated that [the site was going down](https://github.com/JavascriptBattle/javascript-battle-website/issues/345).

Since then (early Oct. 2015) I've been working with @natedsaint to bring it back. We've spent the past year (on and off) tweaking the site, making the game more efficient to run, faster to load, easier to maintain, and giving it a new (but familiar) look.

In short, please join us in bringing back some of the fun that existed with the old site. Bring back your hero, destroy your enemies, and have a blast.

_Sincerely,
John_

_Note: This message is only being sent to users that forked the original "hero-starter" repo._

/cc @Justin0520 @mikemols @KapitanOwies @jaysmoon @benasradzevicius9404 @Vladysanta @unreturnable @rjames93 @mxalfrod @liam-hall @nascarjake @nklunder @Polyrhythm @JohnnySantiagoJr @fabioguarrasi @PMPepper @stuarteske @j-meyer @Tony-Stark @user464091 @ryanhellerud1 @cspy @JesseBlue-ish @jkrehm @pollardld @johnjenkins @TaylorHuston @kdulep @orlando @zoibot @ronfe @kaithrogers @michalweiser @Tempus35 @dukec3 @pgedgaud @rolandbobek @ripptorr @bluemini @Pantazi14 @rnwightman @stalleyg @genesiscreation7 @7Ds7 @wrietersco @jdrucza @ercanezin @DerpingSniper @riahk
